demisto_sdk/commands/common/tests/script_test.py

```diff
@@ -577,3 +577,81 @@ def test_is_native_image_does_not_exist_in_yml_fail(
             integration_validator.is_nativeimage_key_does_not_exist_in_yml()
             == is_validation_ok
         )
+
+    @pytest.mark.parametrize(
+        "yml_content, use_git, expected_results",
+        [
+            ({"comment": "description without dot"}, False, True),
+            (
+                {
+                    "comment": "a yml comment with a dot at the end.",
+                    "args": [
+                        {"name": "test_arg", "description": "description without dot"}
+                    ],
+                },
+                True,
+                False,
+            ),
+            (
+                {
+                    "comment": "a yml comment with a dot at the end.",
+                    "outputs": [
+                        {
+                            "contextPath": "test.path",
+                            "description": "description without dot",
```

Comment:
@YuvHayun you need to add cases where the line ends with a URL (i.e: https://github.com/demisto/content/blob/c918dd8e7dc751436f5359089191b25b09fa17b2/Packs/AnsibleAlibabaCloud/Integrations/AnsibleAlibabaCloud/AnsibleAlibabaCloud.yml#L134)